libraries/Bluefruit52Lib/src/services/BLEHidAdafruit.cpp

```diff
@@ -248,16 +246,10 @@ bool BLEHidAdafruit::mousePan(uint16_t conn_hdl, int8_t pan)
  *------------------------------------------------------------------*/
 bool BLEHidAdafruit::gamepadReport(uint16_t conn_hdl, hid_gamepad_report_t* report)
 {
-  if ( isBootMode() )
-  {
-    return bootGamepadReport(conn_hdl, report, sizeof(hid_gamepad_report_t));
-  }else
-  {
-    return inputReport(conn_hdl, REPORT_ID_GAMEPAD, report, sizeof(hid_gamepad_report_t));
-  }
+   return inputReport(conn_hdl, REPORT_ID_GAMEPAD, report, sizeof(hid_gamepad_report_t));
```

Comment:
`isBootMode()` should still be checked and return false if it is the case. The bootmode() is common characteristic to the whole ble hid device (mouse + keyboard + gamepad). However only mouse + keyboard can send their boot report via boot endpoint.

```
if ( isBootMode() ) return false;
return inputReport(conn_hdl, REPORT_ID_GAMEPAD, report, sizeof(hid_gamepad_report_t));
```